FinniversKit/Sources/Recycling/GridViews/Markets/GridView/Cell/MarketsGridViewCell.swift

```diff
@@ -44,6 +44,7 @@ class MarketsGridViewCell: UICollectionViewCell {
         imageView.setContentCompressionResistancePriority(.defaultLow, for: .vertical)
         imageView.setContentHuggingPriority(.defaultHigh, for: .vertical)
         imageView.contentMode = .scaleAspectFit
+        imageView.tintColor = UIColor(red: (247/255.0), green: (81/255.0), blue: (89/255.0), alpha: 1.0)
```

Comment:
we can't hardcode the brand colour in the sdk, it has to be provided and brand specific

Reply:
one way to do it which does not require a lot of changes could be to set the `tintColor` 
like this:
`collectionView.tintColor = UIColor(red: (247/255.0), green: (81/255.0), blue: (89/255.0), alpha: 1.0)`

in `ToriMarketsDemoView`